.github/workflows/vale.yml

```diff
@@ -3,8 +3,7 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/*.adoc'
-      - '.github/workflows/vale.yml'
```

Comment:
If not absolutely required, I'd like to keep this in to ensure vale runs if we change the vale GHA itself.

Reply:
Not sure I understand. You want me to revert these lines? I thought that they were causing the check not to run.

Reply:
Only line 7; I'd always run an action if the action itself is changed.